packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -49,7 +50,8 @@ export default function Taglist(props) {
   const [ isDropdownExpanded, setIsDropdownExpanded ] = useState(false);
   const [ hasOptionsLeft, setHasOptionsLeft ] = useState(true);
   const [ isEscapeClosed, setIsEscapeClose ] = useState(false);
-  const searchbarRef = useRef();
```

Comment:
You probably have to rebase to `develop` as we used this one for the new `skipLink` to the search bar (cf. #838).